filesystems/http/src/main/scala/cromwell/filesystems/http/HttpPathBuilder.scala

```diff
@@ -53,4 +53,15 @@ case class HttpPath(nioPath: NioPath) extends Path {
   override def pathAsString: String = nioPath.toString.replaceFirst("/", "//")
 
   override def pathWithoutScheme: String = pathAsString.replaceFirst("http[s]?://", "")
+
+  def fetchSize(implicit executionContext: ExecutionContext, actorSystem: ActorSystem): Future[Long] = {
```

Comment:
I couldn't directly override `size` because it's final in `BetterFileMethods`. I ended up deciding I like this separate method better anyway - it's a totally different operation than reading a file attribute, so I think it's helpful that it needs to be called deliberately and the caller needs to handle the `Future`.